DEV.md

```diff
@@ -1,70 +1,22 @@
-## Setting up Maven
+## Setting up Gradle
 
-### Setting up Maven for Forgerock
-
-Download settings.xml file from ForgeRock backstage after you've logged in `http://maven.forgerock.org/repo/private-releases/settings.xml` and put it in `~/.m2/settings.xml`
-
-### Setting up Maven for Trusona
-
-1. Create a master password in Maven:
+1. If it does not already exist, create a `.gradle` directory in your home directory like this:
 
 ```bash
-$ mvn --encrypt-master-password $master_password
-{hfVXK9Wxn+kH0/fzpwehZboNEgLI=}
-```
-
-2. Create `~/.m2/settings-security.xml` with the following:
-
-```xml
-<settingsSecurity>
-  <master>{hfVXK9Wxn+kH0/fzpwehZboNEgLI=}</master>
-</settingsSecurity>
+mkdir -p ~/.gradle
 ```
 
-3. Encrypt your Artifactory password:
+2. Create a file named `gradle.properties` at `~/.gradle/gradle.properties` with the following content:
 
-```bash
-$ mvn --encrypt-password $artifactory_password
-{XuPqXqg2xkgH8a1yPGGznvpEbQuPgaJvg9wXPE2Zytsi8xJbim5KoK3Juk7SULpA==}
+```text
+ARTIFACTORY_USERNAME=<YOUR_ARTIFACTORY_USERNAME>
+ARTIFACTORY_PASSWORD=<YOUR_ARTIFACTORY_USERNAME>
+FORGEROCK_MAVEN_USERNAME=<YOUR_FORGEROCK_MAVEN_USERNAME>
+FORGEROCK_MAVEN_PASSWORD=<YOUR_FORGEROCK_MAVEN_PASSWORD>
+CONTEXT_REPO_ROOT=https://trusona.jfrog.io/trusona/
```

Comment:
Does the `CONTEXT_REPO_ROOT` need to be configurable? Seems like we could at least default it and have one less environment variable to set.

Reply:
I have dropped that configuration